src/routes/RemediationDetailsComponents/SystemsContent/Columns.js

```diff
@@ -0,0 +1,55 @@
+import { wrappable } from '@patternfly/react-table';
+import { renderComponent } from '../helpers';
+import {
+  ActionsCell,
+  NameCell,
+  TagsCell,
+  RebootRequiredCell,
+  OSCell,
+  ConnectionStatusCell,
+} from './Cells';
+
+export default [
+  {
+    title: 'Name',
+    transforms: [wrappable],
+    // sortable: 'action',
```

Comment:
Are we planning to enable sorting? What are the current constraints?

Reply:
Currently, in the old version of things the FE does all the sorting, there is work being done on the BE to imlpement sorting properly, once thats added, we should just be able to uncomment these bits in the columns and sorting should just work